mirgecom/viscous.py

```diff
@@ -0,0 +1,402 @@
+r""":mod:`mirgecom.viscous` provides helper functions for viscous flow.
+
+Viscous Flux Calculation
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. autofunction:: viscous_flux
+.. autofunction:: viscous_stress_tensor
+.. autofunction:: diffusive_flux
+.. autofunction:: conductive_heat_flux
+.. autofunction:: diffusive_heat_flux
+.. autofunction:: viscous_facial_flux
+
+Viscous Time Step Computation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. autofunction:: get_viscous_timestep
+.. autofunction:: get_viscous_cfl
+.. autofunction:: get_local_max_species_diffusivity
```

Comment:
`get` vs not? (for consistency with the above)

Reply:
#485

Reply:
These aren't part of the EOS though.